MapboxNavigation/FeedbackViewController.swift

```diff
@@ -71,7 +75,33 @@ class FeedbackViewController: UIViewController, DismissDraggable, UIGestureRecog
         return fullHeight
     }
     
-    override func viewDidLoad() {
+    var eventsManager: EventsManager
```

Comment:
This property (or at least its getter) should be public. If application code can pass an EventsManger into the initializer, it should be able to see which EventsManager the object wound up with.